presto-tests/src/test/java/com/facebook/presto/tests/TestTpchDistributedStats.java

```diff
@@ -30,11 +28,7 @@ public TestTpchDistributedStats()
     @Test
     void testEstimateForSimpleQuery()
     {
-        String queryId = executeQuery("SELECT * FROM NATION");
-
-        QueryPlan queryPlan = getQueryPlan(queryId);
-
-        MetricComparison rootOutputRowCountComparison = getRootOutputRowCountComparison(queryId, queryPlan);
-        assertEquals(rootOutputRowCountComparison.result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM NATION").result(), MATCH);
```

Comment:
I think this commit should go first, before the superclass / superinterface extraction